src/components/sidebar/right-sidebar/new-datepicker/NewDateSlider.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from "react";
+import { Slider, Rail, Handles, Tracks } from "react-compound-slider";
+import { Handle, Track, TooltipRail } from "./components"; // example render components - source below
+import {useEffect, useState} from "react";
+
+const sliderStyle: React.CSSProperties = {
+  position: "relative",
+  width: "80%",
+  margin: "5% 10%"
+};
+
+interface DateSliderProps {
+  minPossibleValue: number; //minimum possible value for slider
```

Comment:
So minPossibleValue and maxPossibleValue are unix timestamps right? And values are offsets (in terms of milliseconds) from the min date?

Can we comment that explicitly in the code? It's a bit confusing to interpret otherwise

@annoyingpr @HimanshuRanka

Reply:
Yeah you got that right. I can add it in